api-reference/api-introduction/api-keys.md

```diff
@@ -1,10 +1,26 @@
 # API keys
 
-Your API requests are [authenticated](authentication.md) using the account's API key. If you do not include your key when making an API request or use an incorrect one, we return an error.
+ImageKit [authenticates](authentication.md) your API requests using your account’s API keys. If you don’t include your key when making an API request, or use an incorrect or outdated one, ImageKit returns a `403 - Forbidden` response.
+
+The list of APIs whose requests can be authenticated using API keys can be found [here](README.md).
+
+{% hint style="info" %}
+**API key limits**\
+
+- You can only have five active API key pairs at once.
+- Your account will have at least one standard API key pair at all times.
+  {% endhint %}
+
+## Types of keys
+
+- Standard keys: A standard API key has read and write access to all the APIs listed above.
+- Restricted key: A restricted API key allows only the minimum level of access that you specify across all the APIs listed above. The three access levels are: `None`, `Read only` and `Read and write`. For example, if you set `Read only` permission for Media access for your restricted keys, you will not be able to perform any uploads, updates, deletes, or such operations and will only be able to perform operations like list and search files, get file details, get file metadata etc.
```

Comment:
Add a hyperlink to detailed resource access table here

Reply:
Done